### ISSUE TITLE
Nothing Happens

### ISSUE THREAD

**Issue by subzerohurley** [HolographicWings/LethalSDK-Unity-Project#2]
Followed the written tutorial as close as possible, once I go to the "LethalSDK" tab and build it. 
Only for nothing to be in the folder. it makes the "AssetBundles" folder, loads something, but produces nothing. 
For more context, I'm only adding 1 scrap, everything is set up as such too.

**Comment by HolographicWings** [comment]
> Followed the written tutorial as close as possible, once I go to the "LethalSDK" tab and build it. Only for nothing to be in the folder. it makes the "AssetBundles" folder, loads something, but produces nothing. For more context, I'm only adding 1 scrap, everything is set up as such too.

did you defined the proper assetbundle name to every file of your mod ?

**Comment by subzerohurley** [comment]
Yes, I havent tried the recent update tho so i'll give it another shot

**Comment by subzerohurley** [comment]
Yeah the latest version just doesnt work at all for me, the scene is gray, cant edit materials even after extracting them, etc 
Trouble shooting is easier if there was a discord, hence is why I asked about it 😅

![UNITY](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/18029175/d474bba3-7f46-46b1-8634-da32007ebe29)

**Comment by subzerohurley** [comment]
Yeah, it still seems to build nothing, even on the newest version, also on the newest version the skybox/lighting is all dark.
Would be great to have the video tutorial or a discord chat where feedback/trouble shooting would be faster/smoother

**Comment by HolographicWings** [comment]
I will make that tutorial, can you show me a video when you try to create the bundle ? On your video show me the modmanifest of your mod before to click on the build button

**Comment by subzerohurley** [comment]
So the latest version seemed to have fixed it. 
That said I'm at a new issue, tho this is more user error on my part since I'm new to unity. 
I linked the .smd to the "prefab" but it's INCREDIBLY small, is there a way I can link the actual imported model instead? (The icon with the box with 2 blue sides)
![image](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/18029175/44ae1d9c-2956-4743-a944-0a7eb6554951)

**Comment by HolographicWings** [comment]
> So the latest version seemed to have fixed it. That said I'm at a new issue, tho this is more user error on my part since I'm new to unity. I linked the .smd to the "prefab" but it's INCREDIBLY small, is there a way I can link the actual imported model instead? (The icon with the box with 2 blue sides) ![image](https://private-user-images.githubusercontent.com/18029175/289402631-44ae1d9c-2956-4743-a944-0a7eb6554951.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyODI1NDYsIm5iZiI6MTcwMjI4MjI0NiwicGF0aCI6Ii8xODAyOTE3NS8yODk0MDI2MzEtNDRhZTFkOWMtMjk1Ni00NzQzLWE5NDQtMGE3ZWI2NTU0OTUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDA4MTA0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjMTYxMGM5MmMwMjEyNzc0M2JkMGRlMmQ2M2MwYmJkZTllYzEwMzkwNDU2NmIxZWE1MmIyYjg4YjIxZmQ4MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CnKfG9C4i7kfGypLgtZ8qfxQxgfg4077HABYTrmF7nU)

Hello
Changing the prefab scale doesn't work ?
![image](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/19623115/a01decd1-35bd-487a-8fe5-e943ce88f07e)

**Comment by subzerohurley** [comment]
It did! Sorry about that! 
Also outside of changing it through the (Scrap) file, is there a more accurate way to adjust the rotations?
It seems to be completely based off the mesh properties and not the imported model properties

**Comment by HolographicWings** [comment]
> It did! Sorry about that! Also outside of changing it through the (Scrap) file, is there a more accurate way to adjust the rotations? It seems to be completely based off the mesh properties and not the imported model properties

in the prefab too ?
otherwise in blender you can change the default rotation, position and scale, but i don't remember how

**Comment by HolographicWings** [comment]
> It's a bit difficult to explain, It's oriented the way i want it in blender, and when I drag the model into unity, it keeps that, it's the .mesh that isnt keeping the orientation, which is what the the file uses for it's resting place, Tho I'm mainly happy the bundles are building now!

it's hard to see your issue without a video or screenshots
the prefab is made to choose the position of the item you want even if the mesh have wrong values

**Comment by subzerohurley** [comment]
I'll wait on that for when you open a discord, also did you move the module folder? because the mod isnt generating one

**Comment by subzerohurley** [comment]
Ah i see, you changed the modules extensions, along with modules in general, maybe a good idea to update this SDK to convey that.

**Comment by HolographicWings** [comment]
> Ah i see, you changed the modules extensions, along with modules in general, maybe a good idea to update this SDK to convey that.

It's already done, the SDK is supposed to add the extensions automatically

**Comment by subzerohurley** [comment]
Sweet! I love how much work and thought you're putting into this! 
Tho I did notice there's no releases anymore

**Comment by HolographicWings** [comment]
> Sweet! I love how much work and thought you're putting into this! Tho I did notice there's no releases anymore

What do you mean by No releases anymore ?

**Comment by subzerohurley** [comment]
Oh! Fair! My apologies!
